.env.example

```diff
@@ -0,0 +1,4 @@
+# Set up path to Moodle code
+MOODLE_DOCKER_WWWROOT=/path/to/moodle/code
+# Choose a db server (Currently supported: pgsql, mariadb, mysql, mssql, oracle)
+MOODLE_DOCKER_DB=pgsql
```

Comment:
These need an export before them if you want to source them from the scripts. dotenv files like you know them from php, python etc will not work with shells.